stack/dashboards/deb/debian/preinst

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+# postinst script for wazuh-dashboards
+# Copyright (C) 2021, Wazuh Inc.
+#
+# This program is a free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+
+set -e
+
+#
+# This script is executed in the pre-installation phase
+#
+#   On Debian,
+#       $1=install : indicates an new install
+#       $1=upgrade : indicates an upgrade
+
+# configuration variables
+export NAME=wazuh-dashboards
+export CONFIG_DIR=/etc/"${NAME}"
+export INSTALLATION_DIR=/usr/share/"${NAME}"
+export LOG_DIR=/var/log/"${NAME}"
+
+
+case "$1" in
+    # Debian ####################################################
+    install)
+        # environment configuration
+        if [ ! -d "${INSTALLATION_DIR}" ]; then
+            mkdir -p "${CONFIG_DIR}"
+            mkdir -p "${INSTALLATION_DIR}"
+            mkdir -p "${LOG_DIR}"
+        else
+            rm -rf "${INSTALLATION_DIR}"
+            mkdir -p "${INSTALLATION_DIR}"
```

Comment:
I don't understand this else.

Reply:
It is for the case in which for some reason the directory /usr/share/wazuh-dashboards exists